Core/Frameworks/BaikalAdmin/Controller/Install/Database.php

```diff
@@ -46,12 +44,6 @@ function execute() {
             $this->oModel->set('mysql_username', PROJECT_DB_MYSQL_USERNAME);
             $this->oModel->set('mysql_password', PROJECT_DB_MYSQL_PASSWORD);
             $this->oModel->set('encryption_key', BAIKAL_ENCRYPTION_KEY);
-
-            if (defined("BAIKAL_CONFIGURED_VERSION")) {
-                $oStandardConfig = new \Baikal\Model\Config\Standard();
-                $oStandardConfig->set("configured_version", BAIKAL_CONFIGURED_VERSION);
-                $oStandardConfig->persist();
-            }
```

Comment:
Was my assumption right that this config init above leads to just parsed/migrated MySQL settings from `config.system.php` being overwritten by parsing the `baikal.yaml` which contains empty MySQL settings?

Reply:
`baikal.yaml` with empty Sqlite/MySQL settings is created during the form submission of the previous step at https://github.com/sabre-io/Baikal/blob/master/Core/Frameworks/BaikalAdmin/Controller/Install/Initialize.php#L76 but with a `configured_version` of 0.7.1 (the default inside the model).

At the next page load https://github.com/sabre-io/Baikal/blob/master/Core/Frameworks/BaikalAdmin/WWWRoot/install/index.php#L84 sees that the yaml is configured with the current Baikal version (0.7.1) but the `INSTALL_DISABLE` file isn't present yet, so we went up in this file.

The change you've highlighted here keeps `configured_version` at 0.7.1 in `baikal.yaml` until we've also written the Sqlite/MySQL settings to `baikal.yaml` (the additions in this commit are inside a `$this->oForm->persisted()` branch). `configured_version` in `baikal.yaml` is then rolled back to the value in `config.system.php` so that https://github.com/sabre-io/Baikal/blob/master/Core/Frameworks/BaikalAdmin/WWWRoot/install/index.php#L84 goes forward.